File: Challenges/Coding World Cup/2023/Day 1 - ENG vs NZ - Jersey Quest.java
import java.util.*;

public class Solution {
    static int minDifference(int[] a) {
        Arrays.sort(a);
        int minDiff = Integer.MAX_VALUE;
        for (int i = 0; i < a.length - 1; i++) {
            minDiff = Math.min(minDiff, a[i + 1] - a[i]);
        }
        return minDiff;
    }
}

File: Challenges/Coding World Cup/2023/Day 1 - ENG vs NZ - Jersey Quest.py
from typing import *

def minDifference(a: List[int]) -> int:
    # Write your code here.
    a.sort()
    m = a[1] - a[0]
    for i in range(0, len(a) - 1):
        m = min(m, a[i + 1] - a[i])
    return m
